analyzers/tests/SonarAnalyzer.UnitTest/Helpers/MethodParameterLookupTest.cs

```diff
@@ -207,6 +207,7 @@ private void InspectTryGetSyntax(MethodParameterLookupBase<TArgumentSyntax> look
 
             private void InspectTryGetSymbol(MethodParameterLookupBase<TArgumentSyntax> lookup, object expectedArguments, TArgumentSyntax[] arguments)
             {
+                Assert.ThrowsException<ArgumentException>(() => lookup.TryGetSymbol(SpecialArgument.Parent, out var parameter));
```

Comment:
Can you make this a test case a standalone test?
Use `SyntaxFactory.SomeSimpleNodeThatIsNotAnArgument()` to create the node.